### DISCUSSION TITLE
Is it intentional that there is no 2.1.0 release for kairos-io/provider-kairos?

### DISCUSSION THREAD

**Discussion by sdwilsh** [kairos-io/kairos#1418] [Q&A] [answered]
I had noticed the [2.1.0 release for kairos/kairos](https://github.com/kairos-io/kairos/releases/tag/v2.1.0), but then found that there was no 2.1.0 to upgrade to when I ran `kairos-agent upgrade list-releases`.  Historically, `kairos-io/provider-kairos` has had a 1:1 mapping of releases, at least in my experience, so I wasn't sure if this was a bug and I should file an issue or if it was intentional.

**Comment by mauromorales** [comment, marked as answer]
For the Kairos provider, we'll have the release of 2.1.0 some time this week. 

For the agent, yes it is intentional, we just extracted it from the main kairos repo, and we've decided we want to have different versions and release cadence.
